client/ayon_core/plugins/publish/extract_burnin.py

```diff
@@ -378,6 +378,7 @@ def main_process(self, instance):
                 # Prepare subprocess arguments
                 args = list(executable_args)
                 args.append(temporary_json_filepath)
+                args.append("--headless")
```

Comment:
Can we add a comment to this? I feel like it's a bit unclear in the future what this `--headless` is trying to do. Is it a flag for ffmpeg? or is it a flag for how we run openpype processes? (It's unclear to me which).

If it's one for the OpenPype runner logic then I feel this shouldn't be an arg to the `run_ayon_launcher_process` but maybe a clear argument one can pass to `run_ayon_launcher_process` to differentiate what is essentially arguments for the command you are running an which are just 'settings' for the ayon process launcher.

If this `--headless` is an FFMPEG argument, then ignore me.

Reply:
well, docstring for `run_ayon_launcher_process` says:

```
Args:
        *args (str): ayon-launcher cli arguments.
        **kwargs (Any): Keyword arguments for subprocess.Popen.
```
In this regard, `--headless` fits to the `*args` definition. It is indeed **ayon-launcher** flag, not related to ffmpeg.

Reply:
I would just define it here https://github.com/ynput/ayon-core/pull/346/files#diff-04d034303d010fab8f908ef1699037a6087f7c85b41b0d279267c48fa958d654R216 instead of appending if all the time.

Reply:
> clear argument one can pass to run_ayon_launcher_process to differentiate what is essentially arguments for the command you are running an which are just 'settings' for the ayon process launcher.

I don't see any advantage in that.

Reply:
> well, docstring for `run_ayon_launcher_process` says:
> 
> ```
> Args:
>         *args (str): ayon-launcher cli arguments.
>         **kwargs (Any): Keyword arguments for subprocess.Popen.
> ```
> 
> In this regard, `--headless` fits to the `*args` definition. It is indeed **ayon-launcher** flag, not related to ffmpeg.

Actually - seeing it like that makes much more sense. It's  just an argument to AYON; not to FFMPEG.